### ISSUE TITLE
depth loss and point loss

### ISSUE THREAD

**Issue by xiaoxiaoqiang9527** [facebookresearch/vggt#225]
Hello, your superior work is much appreciated.

I am trying to fine-tune VGGT with my own data, but I am getting negative values when calculating the loss. The maximum confidence value calculated is 261.7603 and the mean value is 43.6214. 

I would like to know
1. what is the reasonable range of confidence level?
2. is the default expp1 version of confidence calculation used in training VGGT? Can it be replaced with the sigmoid version?
3. maybe the confidence part is correct and it is the regression loss that is miscalculated?

Very much looking forward to your clarification.

**Comment by jytime** [comment]
1. It is common for the loss to be negative. The confidence value looks a bit high and not too much.
2. In theory it can be replace by sigmoid although I never tried it before.
3. Don't worry, it is correct. You can also try to adjust the alpha value if you hope to suppress the confidence
